apps/hash-frontend/src/lib/user-and-org.ts

```diff
@@ -66,202 +82,216 @@ export const constructMinimalUser = (params: {
   };
 };
 
-export type User = MinimalUser & {
-  memberOf: Org[];
+export type Org = MinimalOrg & {
+  hasAvatar?: {
+    linkEntity: LinkEntity;
+    imageEntity: Image;
+  };
+  memberships: {
+    linkEntity: Entity<OrgMembershipProperties>;
```

Comment:
Maybe we should update the `LinkEntity` type to accept a generic as well, so we can use it here:
```suggestion
    linkEntity: LinkEntity<OrgMembershipProperties>;
```